content/posts/newsletter/2020-09-06.md

```diff
@@ -0,0 +1,134 @@
+---
+title: XMPP bites into the apple!
+save_as: 2020/09/newsletter-09-september/index.html
+url: 2020/09/newsletter-09-september/
+date: 2020-09-06 00:00
+author: emus
+blog_id: newsletter
+category: Newsletter
+---
+
+Welcome to the XMPP newsletter covering the month of September 2020.
+
+### Newsletter translations
+
+Translations of the XMPP Newsletter will be released here:
+
+- French on [linuxfr.org](https://linuxfr.org/tags/xmpp/public) and [jabberfr.org](https://news/jabberfr.org/category/newsletter/)
+
+## XSF Announcements
+
+The XSF started the application period for its annual XSF Board and XSF Council elections. Applicants for the council must be elected XSF members, while board seats are open to everyone. If you are interested to apply for any of those positions please visit the [application page](https://wiki.xmpp.org/web/Board_and_Council_Elections_2020).
+
+The Q4 2020 XSF membership application period is open as well. If you are interested to become an XSF member you can [apply here](https://wiki.xmpp.org/web/Membership_Applications_Q4_2020).
+
+## Articles
+
+The German blog kuketz-blog.de published [part 6 of its messenger series](https://www.kuketz-blog.de/conversations-messaging-ueber-das-xmpp-protokoll-messenger-teil6/) covering the Android client Conversations.
+
+## Software news
+
+### Clients and applications
+
+A new major version of Movim is out. With improved notifications, new video-conferencing and chat features and some major performance overhaul. A new version for the Android application is also available to integrate all those features in the Android environment. Discover all the improvements of this new release on the official release note [Movim 0.18 – Oterma](https://nl.movim.eu/?node/pubsub.movim.eu/Movim/11655111-e7ad-4e0c-975c-3c78755d22aa). Have a look at Movim’s improved draw widget:
+
+![Movim](/images/newsletter/september2020/movim.png "Movim")
+
+[Gajim Development News (September)](https://gajim.org/post/2020-09-27-development-news-september/): This month’s post marks one year of Gajim Development News! The rework of Gajim’s Preferences window has been completed. We improved group chat invitations and made group chat settings easier to access. Also, joining a group chat is now much prettier.
```

Comment:
"We improved [...]" convert it into something like "Gajim has improved [...]" maybe? :)